arangosh/Import/ImportFeature.cpp

```diff
@@ -222,6 +227,14 @@ void ImportFeature::validateOptions(
       FATAL_ERROR_EXIT();
     }
   }
+  for (std::string& str : _removeAttributes) {
+    StringUtils::trimInPlace(str);
+    if (str.empty()) {
+      LOG_TOPIC(FATAL, arangodb::Logger::FIXME)
+        << "cannot remove empty attribute '" << str << "'";
```

Comment:
I think `str` will always be an empty string here... Why print it?

Reply:
Good point, I changed it